BlueprintUI/Sources/Internal/OSLog+Blueprint.swift

```diff
@@ -12,6 +12,20 @@ extension OSLog {
 /// Namespace for logging config.
 public enum BlueprintLogging {
 
+    /// Configuration for logging options
+    public struct Config {
+
+        /// When `true`, measurement data will be collected for all `Element`s
+        public var recordMeasurePasses: Bool
```

Comment:
Maybe this would be better as an enum?

```
enum LogLevel {
    case fullHierarchy
    case completeTimeOnly
}
```
?

Reply:
that seems reasonable, but since at the moment it just governs whether to emit the element measurement logs, a bool seems sufficient.

Reply:
Got it, sounds good then